lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,15 +839,75 @@ def upload_reads(self, ctx, params):
         self.log('Starting upload reads, parsing args')
```

Comment:
upload_reads is now about 100 lines and > 2 pages of code, which is > 2x too long. Should separate into subroutines.